src/start/dropping-packets.md

```diff
@@ -41,7 +41,7 @@ We'll then call `block_ip` to determine the fate of the packet:
 
 In order to add addresses to block, we first need to get a reference to the `BLOCKLIST` map.
 Once we have it, it's simply a case of calling `blocklist.insert()`.
-We'll use the `IPv4Addr` type to represent our IP address as it's human-readable and can be easily converted to a `u32`. We'll block all traffic to `1.1.1.1` for this example.
+We'll use the `IPv4Addr` type to represent our IP address as it's human-readable and can be easily converted to a `u32`. We'll block all traffic **to** `192.168.0.10` sent over the `lo` interface, for this example.
```

Comment:
I'm confused to the rationale of using `lo0`

The whole idea of providing the `--iface` flag was that the interface is configurable - e.g if you are using a VM and you don't have stable interface naming you could use `ens5` or whatever the OS decides you NIC is... or in my case, I used my wireless interface. `wlp2s0`. As you've somewhat eluded to here (by having to use `curl --interface lo`) there isn't much of interest happening on that interface anyway.

If it is not clear that the interface is configurable, I would suggest that we add a section at the beginning - perhaps in the environment setup or prerequisites.

something like;

```
In this example you will be attaching XDP programs to a network interface. The guide assumes you have an interface called `eth0`, but you may select any NIC on your machine.
```

This could be called out in the same block as the `XDP_FLAGS` note.